packages/fxa-profile-server/lib/routes/avatar/default.js

```diff
@@ -19,15 +25,20 @@ module.exports = {
       .response(
         `<svg viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg">
         <defs>
+          <style>
+          @font-face{
+            font-family:"Inter";
+            src:url(data:application/font-woff;charset=utf-8;base64,${font}) format("woff");
```

Comment:
In my local testing the new image is 150KB+ verses 1KB. Is there a different font or some optimization that could be done?

Reply:
i just pushed an optimized font that's ~5kb